packages/@okta/vuepress-site/docs/guides/sharing-cert/update-target-app-with-kid/index.md

```diff
@@ -2,14 +2,11 @@
 title: Update the target app to use the new credential
 ---
 
-Update the target app to use the new credential that you just shared.
-When you update the target app with the `kid`, the target app starts
-using the same certificate to sign assertions.
+Update the target app to use the new credential that you just shared. When you update the target app with the `kid`, the target app starts using the same certificate to sign assertions.
```

Comment:
When you update the target app with the source application's key credential ID `kid` 
(trying to match up all these instances of `kid` - whatever works best, but we should be consistent throughout the guide what `kid` means)